CMakeLists.txt

```diff
@@ -70,12 +76,26 @@ if(USE_ASAN)
   link_libraries(asan)
 endif()
 
+# Enable testing if we are building tests
+if(BUILD_TESTS)
```

Comment:
This if already exists in here

Reply:
Indeed, I have turned off tests being extra enabled